actions/workflows/st2_pkg_e2e_test.yaml

```diff
@@ -117,7 +132,10 @@ tasks:
     action: core.remote_sudo
     input:
       hosts: <% ctx().vm_info.private_ip_address %>
+      # If pipe_into_bootstrap_script is not empty, return it + " | "
+      # If pipe_into_bootstrap_script is empty, return an empty string
       cmd: >-
+        <% switch(ctx().pipe_into_bootstrap_script => concat(ctx().pipe_into_bootstrap_script, " | "), true => "") %>
         bash /tmp/st2_bootstrap.sh
         --<% ctx().pkg_env %>
         --<% ctx().release %>
```

Comment:
Feel free to pass that extra U16 flag to script directly without extra workflow logic as our installer is catch-all and won't fail in case of non-existent param provided.

Reply:
Even better!

Reply:
As https://github.com/StackStorm/st2-packages/pull/682 is now merged, you can add `--u16-add-insecure-py3-ppa`